plugins/help/help.py

```diff
@@ -388,7 +388,7 @@ async def add_subcommands(self, group: Union[commands.Group, commands.Cog]):
                 inline=False,
             )
 
-    async def send_bot_help(self) -> None: # pylint: disable=arguments-differ
+    async def send_bot_help(self, mappings) -> None: # pylint: disable=not-accessed
```

Comment:
The pylint rule you're wanting to disable is named 'unused-argument'

I believe 'not-accessed' is a pylance info thing, not from an actual linter

Reply:
Merci :ok_hand:

Reply:
![IMG_20230605_085050](https://github.com/Curiosity-org/Gipsy/assets/29739547/67d766fd-0964-448a-89cb-3ce22a89b1ed)
Il y a également arguments-differ, c'est écrit à la fin du message sur les annotations Github